test/parallel/test-btoa-atob.js

```diff
@@ -15,3 +15,17 @@ throws(() => buffer.btoa(), /TypeError/);
 
 strictEqual(atob(' '), '');
 strictEqual(atob('  YW\tJ\njZA=\r= '), 'abcd');
+
+strictEqual(atob(null), '\x9Eée');
+strictEqual(atob(NaN), '5£');
+strictEqual(atob(Infinity), '"wâ\x9E+r');
+strictEqual(atob(true), '¶»\x9E');
+strictEqual(atob(1234), '×mø');
+strictEqual(atob([]), '');
+strictEqual(atob({ toString: () => '' }), '');
+strictEqual(atob({ [Symbol.toPrimitive]: () => '' }), '');
+
+throws(() => atob(), /ERR_MISSING_ARGS/);
+throws(() => atob(Symbol()), /TypeError/);
+[undefined, false, () => {}, 0, 1, 0n, 1n, -Infinity, [1], {}].forEach((value) =>
```

Comment:
I took @aduh95's suggestion on iterating the different test cases, but I'm wondering whether it would be better if these were on separate lines. When this test fails, it isn't clear which _specific_ test scenario caused the test to fail. On separate lines, a developer would be able to see exactly which case triggered the failure.

Reply:
Let's add some strings whose `length % 4 !== 1` to ensure we indeed don't count ASCII whitespace.
```suggestion
[
  undefined, false, () => {}, {}, [1],
  0, 1, 0n, 1n, -Infinity,
  'a', 'a\n\n\n', '\ra\r\r', '  a ', '\t\t\ta', 'a\f\f\f', '\ta\r \n\f',
].forEach((value) =>
```